src/main/java/ToDos.java

```diff
@@ -0,0 +1,10 @@
+public class ToDos extends Task {
```

Comment:
```suggestion
public class ToDo extends Task {
```
Perhaps the class name can be singular to match with names of other Classes?